### PR TITLE
Token removed

### PR DESCRIPTION
**Pull request by ahmadcitrusbits**
def

### CODE REVIEW

**Comment by KhaliqCitrusbits** [comment]
asdf

File: app.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ app.get('/status', (req, res) => {
 });
 
 app.get('/ENV', (req, res) => {
-  //token
```
Comment (ahmadcitrusbits) [inline app.js:13]: yar ye kia kia hai

File: app.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ app.get('/status', (req, res) => {
...
+  //hellow
   let PORT = process.env.PORT;
   let NAME = process.env.MY_NAME;
```
Comment (ahmadcitrusbits) [inline app.js:15]: shai hai

**Review by ahmadcitrusbits — COMMENTED** [review]
yar ye changes kardy

File: app.js (hunk 1 above)
Comment (ahmadcitrusbits) [reply app.js:13]: ok

**Pull request closed without merging**